test/e2e/e2e.go

```diff
@@ -576,110 +576,6 @@ var _ = ginkgo.Describe("e2e control plane", func() {
 
 		framework.ExpectNoError(<-errChan)
 	})
-
-	ginkgo.Describe("test tainting a node according to its defaults interface MTU size", func() {
```

Comment:
Why not keep similar tests that verify node transition between `Ready`/`NotReady` instead of `Tainted`/`Not Tainted`?